### DISCUSSION TITLE
Support for Java 18?

### DISCUSSION THREAD

**Discussion by bradsmith** [spotbugs/spotbugs#2032] [General]
Update: You can disregard this discussion.  User error: the update in my pom.xml had not taken affect.  Working now with the latest version.

I'm getting the following error when running the eClipse plugin with Java 18: Exception in thread "main" java.lang.IllegalArgumentException: Unsupported class file major version 62

I'm guessing there is currently no support for Java 18.  Any idea when that might be available?

Thanks in advance.

**Comment by JuditKnoll** [comment]
Closing this Discussion as it's an user error.
